local.go

```diff
@@ -245,16 +245,16 @@ func (l *LocalTest) SendTreeNode(proto string, from, to *TreeNodeInstance, msg n
 	return to.overlay.TransmitMsg(onetMsg, io)
 }
 
-// AddPendingTreeMarshal takes a treeMarshal and adds it to the list of the
+// addPendingTreeMarshal takes a treeMarshal and adds it to the list of the
 // known trees, also triggering dispatching of onet-messages waiting for that
 // tree
-func (l *LocalTest) AddPendingTreeMarshal(c *Server, tm *TreeMarshal) {
+func (l *LocalTest) addPendingTreeMarshal(c *Server, tm *TreeMarshal) {
```

Comment:
This method is really internal to Overlay and should not be exported. It was in the first place because we had external tests so we needed to export API.